Source/Urho3D/UI/Font.h

```diff
@@ -109,6 +112,8 @@ class URHO3D_API Font : public Resource
     FontType fontType_;
     /// Signed distance field font flag.
     bool sdfFont_;
+    /// Pre-Cache charaters
+    HashMap<unsigned, bool> precacheChars_;

```

Comment:
I don't get when this `HashMap` contains `false`...
Maybe `HashSet` will do?

Reply:
Thanks. You`re right.I will change it.:)